launcher/src/main/java/org/apache/spark/launcher/SparkLauncher.java

```diff
@@ -99,7 +99,7 @@ public class SparkLauncher extends AbstractLauncher<SparkLauncher> {
    * @deprecated use `CHILD_CONNECTION_TIMEOUT`
    * @since 1.6.0
    */
-  @Deprecated
+  @Deprecated(since = "3.2.0")
```

Comment:
Which is better, `since = "3.2.0"`  or `since = "3.2.0"` ? The latter is also used in Spark, such as 

https://github.com/apache/spark/blob/093b7e1846b43273538e9b50898685478ceb6089/streaming/src/main/scala/org/apache/spark/streaming/api/java/JavaStreamingContext.scala#L59-L61

Reply:
I prefer this shorter one.